README.md

```diff
@@ -1 +1,29 @@
-# Contact-Form
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+Setup-and-mobile-version-skeleton
+Contact-form
+
+> This is my first portfolio project in Microverse curriculum.
+
+![screenshot](./app_screenshot.png)
+
+## Built With
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
 ⏏︎ Your `README` File is missing information like:
 - follow intructions on how to intstall the project locally,
 - Your MIT License is linked to another person and doesn't contains information for this project would be nice to fix this.

.

Reply:
Hi, Carlos. Thank you for your suggestions. I am working on them. 

As refers to MIT, I am the author of the project - Vahan Khachvakian, another person is my coding partner and just has contributed to my project, in accordance with the requirements of the assignment.

Reply:
Oh, I guess, you mean right links, I put it in MIT.

Also, I add instructions for local installation.